app/presenters/candidate_interface/application_form_presenter.rb

```diff
@@ -88,6 +88,14 @@ def other_qualification_path
       end
     end
 
+    def volunteering_path
+      if volunteering_completed? || volunteering_added? || [true, false].include?(@application_form.volunteering_experience)
```

Comment:
Feels a bit clunky, but can't find a good way to improve

Reply:
Agreed! Could extract a private method?

Reply:
Feel free but not blocking 👍

Reply:
Since https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/725#discussion_r351285435, this isn't too bad anymore.